recipe/meta.yaml

```diff
@@ -60,7 +60,7 @@ test:
     - jinja2
     # Required to test optional Numba features
     - cffi
-    - scipy >=1.2,<1.3
+    - scipy
```

Comment:
Dropping the version constraint on `scipy` as that should no longer be needed. For more context on why it was added and why it can now be removed please read [this comment]( https://github.com/conda-forge/numba-feedstock/pull/29#issuecomment-506846258 ).